logbook-json/src/main/java/org/zalando/logbook/json/QuotedStringReplacementOperator.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.logbook.json;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+
+import java.util.function.BinaryOperator;
+
+import static org.zalando.logbook.json.PrimitiveJsonPropertyBodyFilter.quote;
+
+@AllArgsConstructor
+@EqualsAndHashCode
+final class QuotedStringReplacementOperator<T> implements BinaryOperator<String> {
```

Comment:
There is no need for this class to be generic, is there?

Reply:
Yeah. Not in current use cases, I guess. It can be removed if you want.